src/Tribe/Attendee_Registration/Template.php

```diff
@@ -262,11 +277,17 @@ public function set_page_content( $query ) {
 		}
 
 		if ( $this->is_main_loop( $query ) ) {
+			global $post;
+			if ( $post instanceof WP_Post && has_shortcode( $post->post_content, "tribe_attendee_registration" ) ) {
```

Comment:
:no_entry_sign: String "tribe_attendee_registration" does not require double quotes; use single quotes instead

<sub>hash: 1a1f2b3be2b8839878d70a1419d6c8f7</sub>

Reply:
```suggestion
			if ( $post instanceof WP_Post && has_shortcode( $post->post_content, 'tribe_attendee_registration' ) ) {
```